config/features.yml

```diff
@@ -203,3 +199,7 @@
   enabled: false
   premium: true
   chatwoot_internal: true
+- name: assignment_v2
+  display_name: Assignment V2
+  enabled: false
+  chatwoot_internal: true
```

Comment:
Please keep the order as before. It changed it would behave differently.

The features should keep the order it was defined for the first time.